.npmrc

```diff
@@ -3,4 +3,4 @@ hoist-pattern[]=postcss # package/vite
 hoist-pattern[]=pug # playground/tailwind: @vue/compiler-sfc
 shell-emulator=true
 auto-install-peers=false
-package-manager-strict=false
\ No newline at end of file
+dedupe-injected-deps=false
```

Comment:
why is this needed now?

Reply:
I'm not sure as @sapphi-red added it above, but I can confirm that things don't work without it. I tried to run the tests locally on my machine without it and I got tons of failures

Reply:
whoa, that setting fixes an issue in vite-plugin-svelte too, where a local test package suddenly was a symlink instead of hard linked. :hearts: